package.json

```diff
@@ -101,6 +101,7 @@
     "js-beautify": "1.9.0",
     "keymaster": "1.6.2",
     "mini-css-extract-plugin": "0.5.0",
+    "minimist": "^1.2.0",
```

Comment:
I've used [mri](https://www.npmjs.com/package/mri) for config.js - doesn't hurt too much to have both, but maybe we can combine them.

Reply:
Done. Removed minimist.